### PR TITLE
Raidboss: Alex gordias fr

### PR DESCRIPTION
**Pull request by VanSlanzar**
- Added Translations and corrections fr
- Sorry for the capital letters removed or added. I try to respect the punctuation of my language and the game, even if I know it, this is useless in certain lines.

### CODE REVIEW

File: ui/raidboss/common_replacement.js [hunk 1]
```diff
@@ -126,6 +126,10 @@ let commonReplacement = {
     cn: '--分裂--',
     ko: '--분열--',
   },
+  '--stun--': {
+    de: '--Betäubung--',
+    fr: '--étourdissement--',
```
Comment (VanSlanzar) [inline ui/raidboss/common_replacement.js:131]: Could I have the translations (ja, cn, ko) @Jaehyuk-Lee, @r4ha

Comment (Jaehyuk-Lee) [reply ui/raidboss/common_replacement.js:131]: yes it's right for Korean.

Comment (r4ha) [reply ui/raidboss/common_replacement.js:131]: Japanese is `ja: '--スタン--',`.

Comment (VanSlanzar) [reply ui/raidboss/common_replacement.js:131]: Thank you.

File: ui/raidboss/data/03-hw/raid/a2s.js [hunk 2]
```diff
@@ -154,18 +154,18 @@
...
-        'Giant Bomb': 'Atout',
-        'Gordian Hardhelm': 'casque-dur gordien',
-        'Gordian Hardmind': 'cerveau-dur gordien',
-        'Gordian Sniper': 'sniper gordien',
-        'Gordian Soldier': 'soldat gordien',
-        'Hangar 12': 'Grand hangar GH-12',
-        'Jagd Doll': 'poupée jagd',
-        'King Gobtank G-IV': 'gobtank G-IV roi',
-        'Magitek Gobwidow G-IX': 'gobmygale magitek G-IX',
+        'Giant(?! Bomb)': 'Bombe géante',
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/raid/a2s.js:160]: I think this should be `'Giant Bomb'` and `'Bomb'` should be `'(?!Giant )Bomb'`

Comment (VanSlanzar) [reply ui/raidboss/data/03-hw/raid/a2s.js]: I trust you.

Comment (VanSlanzar) [reply ui/raidboss/data/03-hw/raid/a2s.js:160]: What I did worked after testing games. I tried what you told me, but it gives an error. so I reinstated.
a2s

Comment (quisquous) [reply ui/raidboss/data/03-hw/raid/a2s.js:160]: `'Giant(?! Bomb)': 'Bombe géante'` will translate "Giant Bomb" into "Giant Bombe géante", which seems like not what you want.

In that case, `'Giant Bomb'` should just be `'Giant Bomb'` and `'Bomb'` should be `'(?<!Giant )Bomb(?!e)'`

Comment (VanSlanzar) [reply ui/raidboss/data/03-hw/raid/a2s.js:160]: Oh ok. Thank you.

Comment (VanSlanzar) [reply ui/raidboss/data/03-hw/raid/a2s.js]: `(?<! )Bomb(?!e)': 'Bombe',`
Wouldn't it be better?

Comment (quisquous) [reply ui/raidboss/data/03-hw/raid/a2s.js]: Sure, I think that's equivalent.

**Pull request merged**